# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../m4/hotfix.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- m4-1.4.9/lib/sys_time_.h.vanilla	2007-04-09 16:37:06.000000000 +0000
+++ m4-1.4.9/lib/sys_time_.h	2007-04-09 16:37:57.000000000 +0000
@@ -39,7 +39,7 @@
 #if 1
 # undef gettimeofday
 # define gettimeofday rpl_gettimeofday
-int gettimeofday (struct timeval *restrict, void *restrict);
+int gettimeofday (struct timeval *res, void *res2);
 #endif
 
 #endif /* _gl_SYS_TIME_H */
--- m4-1.4.9/lib/gettimeofday.c.vanilla	2007-04-09 16:38:14.000000000 +0000
+++ m4-1.4.9/lib/gettimeofday.c	2007-04-09 16:39:06.000000000 +0000
@@ -98,7 +98,7 @@
    causes problems.  */
 
 int
-rpl_gettimeofday (struct timeval *restrict tv, void *restrict tz)
+rpl_gettimeofday (struct timeval *tv, void *tz)
 {
 #undef gettimeofday
 #if HAVE_GETTIMEOFDAY
